resources/definitions/ultimaker_method_base.def.json

```diff
@@ -395,25 +396,30 @@
         "speed_wall_0": { "value": "speed_wall * 30/40" },
         "speed_wall_x": { "value": "speed_wall" },
         "support_angle": { "value": 40 },
-        "support_bottom_distance": { "value": "support_z_distance / 2" },
+        "support_bottom_distance": { "value": "layer_height if extruder_nr == support_extruder_nr else 0" },
+        "support_bottom_enable": { "value": "false if extruder_nr == support_extruder_nr else true" },
         "support_bottom_material_flow": { "value": "material_flow" },
+        "support_bottom_wall_count": { "value": "0 if extruder_nr == support_extruder_nr else support_wall_count" },
         "support_brim_enable": { "value": false },
         "support_conical_min_width": { "value": 10 },
         "support_enable": { "value": true },
         "support_extruder_nr": { "value": "int(anyExtruderWithMaterial('material_is_support_material'))" },
-        "support_fan_enable": { "value": false },
+        "support_fan_enable": { "value": "true if extruder_nr == support_extruder_nr else false" },
         "support_infill_rate": { "value": 20.0 },
+        "support_infill_sparse_thickness": { "value": "layer_height" },
         "support_interface_enable": { "value": true },
         "support_interface_material_flow": { "value": "material_flow" },
-        "support_interface_offset": { "value": 0 },
+        "support_interface_offset": { "value": "1 if extruder_nr == support_extruder_nr else 0" },
         "support_interface_pattern": { "value": "'lines'" },
-        "support_interface_wall_count": { "value": 2 },
+        "support_interface_wall_count": { "value": "1 if extruder_nr == support_extruder_nr else 2" },
         "support_material_flow": { "value": "material_flow" },
+        "support_offset": { "value": "1.8 if extruder_nr == support_extruder_nr else 0.8" },
         "support_pattern": { "value": "'lines'" },
+        "support_roof_height": { "value": "4*layer_height if extruder_nr == support_extruder_nr else 5*layer_height" },
         "support_roof_material_flow": { "value": "material_flow" },
         "support_supported_skin_fan_speed": { "value": "cool_fan_speed_max" },
         "support_top_distance": { "value": "support_z_distance" },
```

Comment:
I removed support_z_distance = 0 from the soluble material quality files (which I think were the only redundant values).  Is this the correct resolution for this?

Reply:
It is, and you can also remove it from `ultimaker_method_base.def.json` because this file inherits from `ultimaker.def.json`, which already defines `support_z_distance` to 0 (that is what this specific warning means actually).